.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Recipes added with this pull request (max 5):
+ 
+ - package1
+ - package2
+ 
+I have checked the following in my recipes:
+* [ ] My files end with only 1 newline character, no more, no less.
+* [ ] My recipe follow the same style as [example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml) as much as possible.
+* [ ] The license field also specifies the number of the license if applicable (e.g. `GPLv3` instead of `GPL` or `BSD 3-clause` instead of `BSD`).
+* [ ] My recipe has tests.
+* [ ] I have looked at [pinned packages](https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies) and pinned those packages as stated.
+* [ ] The `summary` just explains the package and does not include the package name. (e.g. Instead of saying `Jupyterhub is a multi-user server for Jupyter notebooks` just say `Multi-user server for Jupyter notebooks`)
+
+If you build for Windows too:
+* [ ] I have read [VC features](https://github.com/conda-forge/staged-recipes/wiki/VC-features) 
+
+If recipe uses make or cmake or ctest:
+* [ ] I have also added `make check` or similar if applicable.
+
+If recipe builds a library:
+* [ ] I have enabled both static and shared libraries.
+
+If recipe builds some C/C++, Fortran or OMP code:
+* [ ] I have not included `gcc` or `libgcc` in `requirements`. Exceptions can be made but must be tested first with `gcc`/`clang` that is already installed in our CI machines.
+
+If it is a Python PyPI package:
+* [ ] I have used `pip` to install it and my recipe has these elements:
```

Comment:
1. `python setup.py install --single-version-externally-managed --record=record.txt`
2. `python setup.py install`

For now, let's document what we do, which is have people try 1, but if they don't use `setuptools` or it fails fallback to 2.